docs/dev/releasing.md

```diff
@@ -28,10 +28,13 @@ This document guides maintainers through the process of creating an Envoy Gatewa
    ```
 
 7. Sign, commit, and push your changes to your fork. Send a PR to get your changes merged into the release branch.
-   Do not proceed until your PR is merged.
-8. Tag the head of your release branch with the release tag. For example:
+   Do not proceed until your PR is merged and the Build and Test [release GitHub action][] has completed for your PR.
```

Comment:
why not move 6. before 4. to achieve the same thing :)

Reply:
Before 4 is pushing to main while this step is meant to work off the release branch and submit a PR to the release branch. Due to https://github.com/envoyproxy/gateway/issues/632 quickstart updates need to be pushed to main so I'll update for now.